app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("All friends should be vaccinated")
+
+        if visitor["vaccine"]["expiration_date"] < datetime.date.today():
+            raise OutdatedVaccineError("All friends should be vaccinated")
+
+        if visitor.get("wearing_a_mask", False) is False:
+            raise NotWearingMaskError("Friends should buy 1 mask")

```

Comment:
The error message for `NotWearingMaskError` could be more descriptive. Consider using a message like 'Visitor is not wearing a mask.' to clearly indicate the issue.